packages/core/review-workflows/admin/src/routes/settings/hooks/useReviewWorkflows.ts

```diff
@@ -31,7 +31,6 @@ const useReviewWorkflows = (params: UseReviewWorkflowsArgs = {}) => {
 
   const { data, isLoading, error } = useGetWorkflowsQuery(
     {
-      populate: ['stages', 'stageRequiredToPublish'],
```

Comment:
not tested yet, but why were we populating these fields in the first place? arent they necessary?

Reply:
Tbh I don't know the original reason, also because we are not doing it for the create and update methods, btw with this array we don't have the correct returning values because what we need to populate the stages with permissions is something like this
`{
  "stages": {
    "populate": {
      "permissions": {
        "fields": [
          "action",
          "actionParameters"
        ],
        "populate": {
          "role": {
            "fields": [
              "id",
              "name"
            ]
          }
        }
      }
    }
  },
  "stageRequiredToPublish": true
}`